public/components/tournament/list/list.html

```diff
@@ -0,0 +1,13 @@
+<div id="main"></div>
+
+<script src="../../../node_modules/steal/steal.js" main="@empty">
+	import "bitballs/models/fixtures/tournaments";
+	import "bitballs/components/tournament/list/";
+	import $ from 'jquery';
+	import stache from "can/view/stache/";
+
+	localStorage.clear();
+	var template = stache('<tournament-list {is-admin}="true"/>');
```

Comment:
I'm happy that this work now.  `null`, `undefined`, `2` would all not be passed as strings with this syntax.